components/drivers/misc/adc.c

```diff
@@ -154,28 +162,42 @@ rt_err_t rt_adc_disable(rt_adc_device_t dev, rt_uint32_t channel)
     return result;
 }
 
-rt_uint32_t rt_adc_voltage(rt_adc_device_t dev, rt_uint32_t channel)
+rt_int16_t rt_adc_voltage(rt_adc_device_t dev, rt_uint32_t channel)
 {
-   rt_uint32_t value = 0, voltage = 0;
+    rt_uint32_t value = 0;
+    rt_int16_t vref = 0, voltage = 0;
+    rt_uint8_t resolution = 0;
 
     RT_ASSERT(dev);
 
+    /*get the convert bits*/
+    if(_adc_control((rt_device_t) dev, RT_ADC_CMD_GET_RESOLUTION, &resolution) != RT_EOK)
+    {
+        goto _voltage_exit;
+    }
+
+    /*get the reference voltage*/
+   if( _adc_control((rt_device_t) dev, RT_ADC_CMD_GET_VREF, &vref) != RT_EOK)
```

Comment:
```suggestion
   if ( _adc_control((rt_device_t) dev, RT_ADC_CMD_GET_VREF, &vref) != RT_EOK)
```

Reply:
Please care about the format. 4 spaces rather than 3 spaces. if + a blank space + (